backend/src/main/java/hanglog/community/service/CommunityService.java

```diff
@@ -0,0 +1,54 @@
+package hanglog.community.service;
+
+import hanglog.auth.domain.Accessor;
+import hanglog.city.domain.City;
+import hanglog.community.domain.repository.LikesRepository;
+import hanglog.community.domain.type.PublishedStatusType;
+import hanglog.community.dto.response.CommunitySingleTripResponse;
+import hanglog.community.dto.response.CommunityTripsResponse;
+import hanglog.trip.domain.Trip;
+import hanglog.trip.domain.TripCity;
+import hanglog.trip.domain.repository.TripCityRepository;
+import hanglog.trip.domain.repository.TripRepository;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Transactional
+public class CommunityService {
+
+    private final LikesRepository likesRepository;
+    private final TripRepository tripRepository;
+    private final TripCityRepository tripCityRepository;
+
+    public CommunityTripsResponse getTripsByPage(final Accessor accessor, final Pageable pageable) {
+        final List<Trip> trips = tripRepository.findPublishedTripByPageable(pageable.previousOrFirst());
+        final List<CommunitySingleTripResponse> communitySingleTripResponses = trips.stream()
+                .map(trip -> getTripResponse(accessor, trip))
+                .toList();
+        final Long lastPageIndex =
+                tripRepository.countTripByPublishedStatus(PublishedStatusType.PUBLISHED) / pageable.getPageSize() + 1;
```

Comment:
```suggestion
final Long lastPageIndex = tripRepository.countTripByPublishedStatus(PUBLISHED) / pageable.getPageSize() + 1;
```

PublishedStatusType static import로 줄이고 한줄로 하는것도 이쁠것 같아요.(그냥 의견임)